src/lib/stream/stream_cipher.cpp

```diff
@@ -24,7 +24,7 @@
   #include <botan/ofb.h>
 #endif
 
-#if defined(BOTAN_HAS_RC4)
+#if defined(BOTAN_HAS_RC4) && !defined(OPENSSL_NO_RC4)
```

Comment:
This is wrong - just building against an OpenSSL with RC4 disabled should not implicitly disable RC4 in the library itself.